.expeditor/buildkite/cypress.sh

```diff
@@ -6,13 +6,21 @@ cd /workdir/e2e
 
 data=$(curl --silent "https://a2-${CHANNEL}.cd.chef.co/assets/data.json")
 instances_to_test=$(jq -nr --argjson data "$data" '$data[] | select(.tags | any(. == "e2e")) | .fqdn')
+versions=(v1 v2 v2.1)
 
 for instance in ${instances_to_test[*]}
 do
   if ! jq -enr --arg fqdn "$instance" --argjson data "$data" '$data[] | select(.fqdn == $fqdn) | .tags | any(. == "saml")'; then
     export CYPRESS_SKIP_SSO=true
   fi
 
+  for version in ${versions[*]}
+  do
+    if jq -enr --arg version "$version" --arg fqdn "$instance" --argjson data "$data" '$data[] | select(.fqdn == $fqdn) | .tags | any(. == $version)'; then
```

Comment:
there's no `version` key for the tags, so we have to check for each tag by value `v1, v2, or v2.1`